app/assets/stylesheets/course.css.scss

```diff
@@ -1,13 +1,15 @@
 .list-group-item.course {
-  min-height:300px;
+  min-height:425px;
 
-  h3 { margin-top:10px; }
+  h3 { margin-top:10px; text-align:center; }
+  .icon img { width:60%; margin:1.5em 20%; }
```

Comment:
Properties should be ordered margin, width<br>Selector should have depth of applicability no greater than 2, but was 3<br>Colon after property should be followed by one space